### PR TITLE
ci: add pep8-naming to flake8 and fix warnings

### PR DESCRIPTION
**Pull request by jon-hagg**
### Purpose
Add pep8-naming check to flake8, so it is run by tox locally and in github. 
See https://github.com/Breakthrough-Energy/RenewableEnergyProject/issues/343 (not sure how else to link this)

### What the code is doing
In this case, we only had to ignore one warning, everything else is just switching to snake_case. 

### Testing
`tox -e flake8`

### Time estimate
5 min

### CODE REVIEW

File: prereise/gather/solardata/nsrdb/sam.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import numpy as np
 import pandas as pd
 import PySAM.Pvwattsv7 as PVWatts
-import PySAM.PySSC as pssc
+import PySAM.PySSC as pssc  # noqa: N813
```
Comment (rouille) [inline prereise/gather/solardata/nsrdb/sam.py:4]: What is the issue here?

Comment (jon-hagg) [reply prereise/gather/solardata/nsrdb/sam.py:4]: We get the following warning:
`prereise/gather/solardata/nsrdb/sam.py:4:2: N813 camelcase 'PySAM.PySSC' imported as lowercase 'pssc'`

**Review by rouille — APPROVED** [review]
Looks good

**Review by danielolsen — APPROVED** [review]
This will be very helpful for our collaborators.

**Pull request merged**